Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -95,7 +95,8 @@ def glyph(self, componentFlags=0x4):
             component.glyphName = glyphName
             if transformation[:4] != (1, 0, 0, 1):
                 component.transform = (transformation[:2], transformation[2:4])
-            component.x, component.y = transformation[4:]
+            component.x = int(transformation[4])
+            component.y = int(transformation[5])
```

Comment:
instead of truncating the float with `int(f)`, I'd rather do `int(round(f))`, no?